src/extension/project.js

```diff
@@ -210,7 +210,7 @@ export function assembleProject({
   } else {
     giturl = `https://github.com/${owner}/${repo}/tree/${ref}`;
   }
-  const id = `${owner}/${repo}/${ref}`;
+  const id = `${owner}/${repo}`;
```

Comment:
This change is not directly related to this issue, but we currently only ever add projects with the `main` ref, so this suffx in the ID is obsolete.